docs/Getting Started/Adapting to the Head Unit Language/index.md

```diff
@@ -105,3 +111,25 @@ public LifecycleConfigurationUpdate managerShouldUpdateLifecycle(Language langua
 ```
 !@
 
+@![javascript]
+```javascript
+_managerShouldUpdateLifecycle (language) {
+    const appName;
+    switch (language) {
+        case Language.EN_US:
+            appName = 'App Name in English';
+            break;
+        case Language.ES_MX:
+            appName = 'App Name in Spanish';
+            break;
+        case Language.FR_CA:
+            appName = 'App Name in French';
+            break;
+        default:
+            return null;
+    }
+
+    return new LifecycleConfiguration().setAppName(appName);
```

Comment:
There's an issue with no `LifecycleConfigurationUpdate` existing and with the public `managerShouldUpdateLifecycle` function not returning anything. May just have to remove this section from the JS guide for now.

Reply:
Since this section relies on the bug being resolved with the `managerShouldUpdateLifecycle` function, and with no `LifecycleConfigurationUpdate`, we should at least state that this isn't currently supported in the current version of the JS suite, and that will be fixed in a future release